### ISSUE TITLE
[Bug Report] V-dialog keydown events doesn't work when embedding expansion panel

### ISSUE THREAD

**Issue by varadekd** [vuetifyjs/vuetify#11427]
### Environment
**Vuetify Version:** 2.2.22
**Vue Version:** 2.6.11
**Browsers:** Chrome 81.0.4044.138
**OS:** Mac OS 10.15.4

### Steps to reproduce
1. Create a v-dialog.
2. Inside the v-dialog use expansion panels
3. add keydown.esc to v-dialog
4. Now open the dialog and close by clicking on esc button (Works fine).
5. Now again open dialog and expand any of the expansion panel, and now try esc button it doesn't close the model. (Note: keydown.esc is not even triggered).

### Expected Behavior
The dialog should close even after actions are performed on expansion.

### Actual Behavior
The dialog is not closing when the expansion panel is expanded.

### Reproduction Link
<a href="https://codepen.io/varadekd/pen/KKdrVmR" target="_blank">https://codepen.io/varadekd/pen/KKdrVmR</a>

### Other comments
Investigating on my end, I found that the function is not getting called when I am pressing esc key. Till this issue is resolved I am using event listener to detect esc key is pressed. Please look into this issue.

**Comment by ElijahKotyluk** [comment]
I think what's going on is when you click on the expansion panel, that expansion panel is focused which isn't listening for the `esc` key. If you click back on the dialog and press `esc` it closes.

**Comment by varadekd** [comment]
Yes this is what is exactly happening. Is this by design the expectation is no matter which child is focused if the parent is calling an event it should get triggered.

**Comment by jacekkarczmarczyk** [comment]
Duplicate of https://github.com/vuetifyjs/vuetify/issues/11257

**Comment by jacekkarczmarczyk** [comment]
Hmmm, not really duplicate

**Comment by ElijahKotyluk** [comment]
That was my thought when I checked the other related issues, so I figured I’d wait on someone to confirm. Seems like a bug. I’ll look into it this evening if it still hasn’t been triaged.

**Comment by varadekd** [comment]
Any updates regarding this?

**Comment by mariobuikhuizen** [comment]
I have the same issue with `v-tabs` in a `v-dialog` (or `v-menu`). After clicking a tab, the <kbd>ESC</kbd> key doesn't close the dialog/menu. If I press <kbd>TAB</kbd> once and then <kbd>ESC</kbd> it **does** close.

If I check `document.activeElement` in the console after clicking a tab, it shows the active element as `body` instead of one of the tabs, so the key event is not reaching the `v-dialog`.

**Comment by mariobuikhuizen** [comment]
I think this is the problem:
https://github.com/vuetifyjs/vuetify/blob/3acc307d52494fb67ff1d43b4868ce475ac3476d/packages/vuetify/src/components/VTabs/VTab.ts#L83

And this for `v-expansion-panel`:
https://github.com/vuetifyjs/vuetify/blob/3acc307d52494fb67ff1d43b4868ce475ac3476d/packages/vuetify/src/components/VExpansionPanel/VExpansionPanel.ts#L72

**Comment by eveningforgery** [comment]
Fell onto this bug, any workaround found yet?

**Comment by KaelWD** [comment]
This is related to #9808 and #9046
